integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/OperatorUtils.java

```diff
@@ -47,6 +47,10 @@
 
 public class OperatorUtils {
 
+  public static final String OPERATOR_VERSION_340 = "3.4.0";
+  public static final String OPERATOR_340_IMAGE_OWLS_99380 =
+      "phx.ocir.io/weblogick8s/weblogic-kubernetes-operator:owls_99380";
```

Comment:
Is this a public image? Is it accessible to anyone without authentication?

Reply:
Yes.